tests/e2e/triggers/workloads.yaml

```diff
@@ -25,7 +25,7 @@ disk_storage_properties_edges_false: &disk_storage_properties_edges_false
     main:
       args: ["--bolt-port", *bolt_port, "--log-level=TRACE", "--also-log-to-stderr", "--storage-properties-on-edges=False"]
       log_file: "triggers-e2e.log"
-      setup_queries: []
```

Comment:
You should add `STORAGE MODE ON_DISK_TRANSACTIONAL` to the disk_template_cluster workload too.

Reply:
It's added in another PR: https://github.com/memgraph/memgraph/pull/1134